bundle/metadata/annotations.yaml

```diff
@@ -1,12 +1,15 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: beta
-  operators.operatorframework.io.bundle.channels.v1: beta
-  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  # Core bundle annotations.
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: mongodb-atlas-kubernetes
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.4.2
+  operators.operatorframework.io.bundle.channels.v1: beta
+  operators.operatorframework.io.bundle.channel.default.v1: beta
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.7.1
```

Comment:
if we are changing the operator version, is that possible to use the latest [version](https://github.com/operator-framework/operator-sdk/tree/v1.9.0) ? WDYT?

Reply:
I do not know. This version is automatically updated by the SDK itself :)